res/designer/sceneeditorwidget.ui

```diff
@@ -90,15 +90,30 @@
          </layout>
         </widget>
        </item>
-       <item>
+       <!-- <item>
         <widget class="QListView" name="graphics_listview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
+            <property name="acceptDrops">
+             <bool>true</bool>
+            </property>
+            <property name="dragEnabled">
+             <bool>true</bool>
+            </property>
+            <property name="dragDropMode">
+             <enum>QAbstractItemView::InternalMove</enum>
+            </property>
+            <property name="defaultDropAction">
+             <enum>Qt::MoveAction</enum>
+            </property>
         </widget>
+       </item> -->
+       <item>
+        <widget class="DraggableListWidget" name="graphics_listWidget"/>
```

Comment:
When I saved it in Designer it added `native="true"` -- I suggest including that so it doesn't get added when next edited:
```
<widget class="DraggableListWidget" name="graphics_listWidget" native="true"/>
```